tests/drop_packets/drop_packets.py

```diff
@@ -42,7 +42,7 @@ def fanouthost(request, duthost, localhost):
         for file_name in os.listdir(os.path.join(os.path.dirname(__file__), "fanout")):
             # Import fanout configuration handler based on vendor name
             if "mellanox" in file_name:
-                module = importlib.import_module("fanout.{0}.{0}_fanout".format(file_name.strip(".py")))
+                module = importlib.import_module("..fanout.{0}.{0}_fanout".format(file_name.strip(".py")), __name__)
```

Comment:
qq: A few lines up we check if the DUT asic_type is `mellanox` - is this correct? Shouldn't we be checking if the fanout we're connected to is an OnyxHost?

Reply:
"A few lines up we check if the DUT asic_type is mellanox - is this correct?" - yes its correct.
"Shouldn't we be checking if the fanout we're connected to is an OnyxHost?" - its enough to check if asic type is mellanox and then create appropriate fanout instance.
Because this line will initialize fanout instance specific for Mellanox:
```
module = importlib.import_module("..fanout.{0}.{0}_fanout".format(file_name.strip(".py")), __name__)
fanout = module.FanoutHandler(duthost, localhost)
```

Reply:
I am still confused how this works. For example, what happens if I have a mellanox DUT that is connected to an EOS fanout?

Reply:
I agree that whoever has the setup to test and debug it should add the fanout handler, but we shouldn't select the fanout handler based on the DUT ASIC. I can try to make this change but it might be hard for me to verify there are no regressions because I don't have an Onyx fanout in my lab setup.

Reply:
Understand. Ok so I will update this part to be based on fanout OS type. Correct?

Reply:
Yep, that's right! Thanks! :)

Do you want to add that on to this PR or would you like me to merge this one first so that the other bug fixes get picked up?

Reply:
Better will be to merge this and I will create issue to fix current behaviour.

Reply:
Sounds good, thanks again!